File: src/weaponsprocurement/gui/StockReviewFormat.java
package weaponsprocurement.gui;

import weaponsprocurement.core.CreditFormat;

final class StockReviewFormat {
    private StockReviewFormat() {
    }

    static String credits(long credits) {
        if (credits == StockReviewQuoteBook.PRICE_UNAVAILABLE) {
            return "?";
        }
        return CreditFormat.credits(Math.abs(credits));
    }
}

File: src/main/kotlin/weaponsprocurement/gui/StockReviewFormat.kt
package weaponsprocurement.gui

import weaponsprocurement.core.CreditFormat
import kotlin.math.abs

class StockReviewFormat private constructor() {
    companion object {
        @JvmStatic
        fun credits(credits: Long): String {
            if (credits == StockReviewQuoteBook.PRICE_UNAVAILABLE.toLong()) return "?"
            return CreditFormat.credits(abs(credits))
        }
    }
}
